pkg/vpcmodel/abstractVPC.go

```diff
@@ -82,6 +81,45 @@ type Node interface {
 	IsPublicInternet() bool
 }
 
+// InternalNodeIntf captures common properties for internal nodes: single IP address
+type InternalNodeIntf interface {
+	// an InternalNodeIntf has an exact one IP Address
+	Address() string
+	IPBlock() *common.IPBlock
+	SetIPBlockFromAddress() error
+}
+
+// InternalNode implements interface InternalNodeIntf
+type InternalNode struct {
+	AddressStr string
+	IPBlockObj *common.IPBlock `json:"-"`
```

Comment:
added

Reply:
The field marked by this attribute - `IPBlockObj` of type `*common.IPBlock`.
It's of type of which JSON format is not supported, so it would just add this field name with curly brackets to the JSON output..

Reply:
// avoid having this field, which is of type not supported by json, in the JSON output <which json output?>